### PR TITLE
removed the deprecated touchaction implementation and replaced with sequence class

### PR DESCRIPTION
**Pull request by amuthansakthivel**
Touch Action class used in the drag and drop method is deprecated in Appium latest version.
Replaced it with Sequence class implementation.

### CODE REVIEW

**Comment by amuthansakthivel** [comment]
@asolntsev @BorisOsipov - Please review and provide your suggestions

**Review by asolntsev — APPROVED** [review]

**Comment by asolntsev** [comment]
@BorisOsipov LGTM

File: src/main/java/com/codeborne/selenide/appium/commands/AppiumDragAndDropTo.java [hunk 1]
```diff
@@ -3,47 +3,58 @@
...
+      .addAction(finger.createPointerMove(ofMillis(0), PointerInput.Origin.viewport(), source.x, source.y))
+      .addAction(finger.createPointerDown(PointerInput.MouseButton.MIDDLE.asArg()))
+      .addAction(new Pause(finger, ofMillis(600)))
+      .addAction(finger.createPointerMove(ofMillis(600),
+        PointerInput.Origin.viewport(),
+        target.x, target.y))
+      .addAction(finger.createPointerUp(PointerInput.MouseButton.MIDDLE.asArg()));
+  }
+
+  public Point getCenter(WebElement element){
```
Comment (BorisOsipov) [inline src/main/java/com/codeborne/selenide/appium/commands/AppiumDragAndDropTo.java:55]: (small note) why is it public?

**Review by BorisOsipov — APPROVED** [review]

File: src/main/java/com/codeborne/selenide/appium/commands/AppiumDragAndDropTo.java (hunk 1 above)
Comment (amuthansakthivel) [reply src/main/java/com/codeborne/selenide/appium/commands/AppiumDragAndDropTo.java]: Fixed it now. Thanks

**Pull request merged**